source/geometries/Next100FieldCage.cc

```diff
@@ -72,8 +72,8 @@ Next100FieldCage::Next100FieldCage(G4double grid_thickn):
   // cathode_sapphire_wdw_dist_(257.8 * mm),
 
   // external to teflon (hdpe + rings + holders)
-  hdpe_tube_int_diam_ (1093. * mm),
-  hdpe_tube_ext_diam_ (1105.4 * mm),
+  hdpe_tube_int_diam_ (1093. * mm - 11. * mm), // we keep the thickness, but we make it smaller to be closer to the staves
+  hdpe_tube_ext_diam_ (1105.4 * mm - 11. * mm),
```

Comment:
I don't get it. The teflon panels shouldn't touch the HDPE tube because they have the holder/staves in between.

Reply:
Sorry, I meant the staves, not the teflon panels.

Reply:
There is an incongruence somewhere, because the 2D drawings say that the internal diameter is 1093. mm. I'll have a look!

Reply:
Here you can find the technical drawing of hdpe, slide 4: 
[NEXT-100_111522_HDPE_and_staves_PLANOS FABRICACIÓN.pdf](https://github.com/next-exp/nexus/files/13635208/NEXT-100_111522_HDPE_and_staves_PLANOS.FABRICACION.pdf)

As you can see, the internal diameter is 1093 mm. However, with that diameter hdpe is not touching the staves, provided that the active diameter is 984 mm (which has not been measured directly and I don't know what the techincal drawings say).

Reply:
If we leave the hdpe diameter as it is in the drawings, the distance would be like in the picture:
<img width="991" alt="Screenshot 2023-12-11 at 14 49 05" src="https://github.com/next-exp/nexus/assets/8114436/e82948f6-e291-4d2c-8d7f-6bafe7dcecc0">

Take into account that hdpe is a cylinder, although in the picture it is shown as a polyhedron.

Reply:
Shall we keep it like this?

Reply:
I realized that the dimensions of the hdpe were those of the original external wrap. I've changed them to those of the internal one, since they are almost touching the staves. I've also updated the distance between two opposite teflon panels, according to the drawings. We know from in-situ measurements that the field cage is not a perfect cylinder, but it is slightly deformed. However, it is impossible to simulate that correctly, therefore we decided to keep the simulation simple.